examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OverviewFragment.kt

```diff
@@ -113,6 +115,27 @@ class OverviewFragment : Fragment(), OfferingCardAdapter.OfferingCardAdapterList
         }
     }
 
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+    override fun onResume() {
+        super.onResume()
+
+        val activity = requireActivity() as MainActivity
+        val deepLink = activity.rcDeepLink ?: return
+        if (deepLink is Purchases.DeepLink.WebRedemptionLink) {
+            activity.clearDeepLink()
```

Comment:
Not a fan of the developer having to handle the deep links like this... I think there is a big chance of the developer passing us the same link multiple times (which could still happen with the approach in #1880 but less likely IMO).

Reply:
Can we keep track of what is already redeemed? E.g. by storing those tokens/urls on disk or something.

Reply:
Yes we could... Though I guess there is also a chance it's a valid use case to redeem, as in, maybe the redemption failed the first time due to a connection error or something and they need to try again. I guess we could only "cache" it if there is a successful redemption in this case, but if we for example add deep links to display the paywall in the future, there would be valid use cases of showing the paywall multiple times.

Not to say it's not ok! This approach does provide much more flexibility to the developer, but also more responsibility :P

Reply:
Yea indeed, I meant to cache successful redemptions. If only to alleviate our own backend. Paywall or other non web purchase deeplinks would indeed be excluded from this.